SpokestackTests/SpokestackSpeechRecognizerTest.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  SpokestackSpeechRecognizerTest.swift
+//  SpokestackTests
+//
+//  Created by Noel Weichbrodt on 7/16/20.
+//  Copyright © 2020 Spokestack, Inc. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import Spokestack
+
+@available(iOS 13.0, *)
+class SpokestackSpeechRecognizerTest: XCTestCase {
```

Comment:
There's a lot more this class could be doing to test authentication errors, multiple requests' reuse of the socket connection, socket errors, and the handling of JSON results and the hypothesis list, but I won't insist on any of it.